MEEW-Client/MEEW-Client/Screens/Select/Controller/SelectVC.swift

```diff
@@ -8,38 +8,57 @@
 import UIKit
 
 class SelectVC: BaseVC {
-
+    
     @IBOutlet weak var selectCV: UICollectionView!
     @IBOutlet weak var backgroundView: UIView!
-    
+  @IBOutlet weak var SelectCVCHeightConstraint: NSLayoutConstraint!{
+    didSet {
+      let screenWidth = UIScreen.main.bounds.width
+      let cellWidth = (290/375) * screenWidth
+      let cellHeight = cellWidth * (366/290)
+      SelectCVCHeightConstraint.constant = cellHeight + 20
+    }
+  }
```

Comment:
개인적으로 이 부분도 함수로 빼놓고 사용하면 가독성이 더 좋지 않을까 싶습니다.
```suggestion
  @IBOutlet weak var SelectCVCHeightConstraint: NSLayoutConstraint!{
    didSet {
      updateConstraint()
    }
  }
```